mintbase-deps/src/impls.rs

```diff
@@ -1732,8 +1749,8 @@ impl MintbaseStore {
     pub fn nft_token(
         &self,
         token_id: U64,
-    ) -> Option<TokenCompliant> {
-        Some(self.nft_token_compliant_internal(token_id.0))
+    ) -> Option<TokenCompliantExternal> {
+        Some(self.nft_token_compliant_internal(token_id.0).to_external())
```

Comment:
re: https://github.com/Mintbase/mintbase-core/pull/11/files#pullrequestreview-884908216

Without `to_external` it is impossible to serialise `Owner` as String. You either need to refactor all contracts or create new struct.